kse/build.gradle

```diff
@@ -509,6 +509,7 @@ ospackage {
 	link('/usr/share/applications/kse.desktop', '/opt/kse/kse.desktop', 0644)
 	for (size in [16, 32, 48, 128, 256, 512]) {
 		link("/usr/share/icons/hicolor/${size}x${size}/apps/kse.png", "/opt/kse/icons/kse_${size}.png", 0644)
+		fileMode 0644
```

Comment:
If your intention was to set the permissions for the linked icons, then this is not the right way. The link command already does that (see 3. parameter).